Demos/02-add-aad-auth/README.md

```diff
@@ -4,7 +4,12 @@ The version of the project in this directory reflects completing the tutorial up
 
 > **Note:** It is assumed that you have already registered an application in the app registration portal as specified in [Register the app in the portal](https://docs.microsoft.com/graph/tutorials/aspnet?tutorial-step=2). You need to configure this version of the sample as follows:
 >
+> 1. Determine your ASP.NET applications's SSL URL. In Visual Studio's Solution Explorer, select the 
```

Comment:
This isn't needed since if the user has downloaded from GitHub they get the port already set in the project.

Reply:
So you don't want this step included, and you want the port hard-coded in the example config?

Reply:
Yes. The only time someone's reading this is if they downloaded the project from GitHub, where the port is fixed.

Reply:
Got it... last commit fixed this.